main.js

```diff
@@ -258,10 +263,118 @@ renderCatalog.addEventListener('click', function (event) {
 function view(viewName) {
   if (viewName === renderCatalog.getAttribute('data-view')) {
     renderDetails.setAttribute('class', 'hidden')
+    renderCart.setAttribute('class', 'hidden')
     renderCatalog.removeAttribute('class', 'hidden')
   }
   else if (viewName === renderDetails.getAttribute('data-view')) {
     renderCatalog.setAttribute('class', 'hidden')
+    renderCart.setAttribute('class', 'hidden')
     renderDetails.removeAttribute('class', 'hidden')
   }
+  else {
+    renderDetails.setAttribute('class', 'hidden')
+    renderCatalog.setAttribute('class', 'hidden')
+    renderCart.removeAttribute('class', 'hidden')
+  }
+}
+
+function cartView(item) {
+
+  var container = document.createElement('div')
+  var row = document.createElement('div')
+  var firstCol = document.createElement('div')
+  var img = document.createElement('img')
+  var secondCol = document.createElement('div')
+  var title = document.createElement('h5')
+  var subTitle = document.createElement('h6')
+  var desc = document.createElement('p')
+  var price = document.createElement('p')
+
+  container.setAttribute('class', 'container')
+  container.setAttribute('id', 'details')
+  container.style.backgroundColor = 'white'
+  row.setAttribute('class', 'row')
+  firstCol.setAttribute('class', 'col col-lg-4')
+  firstCol.style.marginTop = '50px'
```

Comment:
You should add CSS classes to your stylesheet and use them instead of adding the styles directly to your elements.